e2e/consul/consul.go

```diff
@@ -176,7 +176,7 @@ func (tc *ConsulE2ETest) TestCanaryInplaceUpgrades(f *framework.F) {
 
 		return true, nil
 	}, func(err error) {
-		t.Fatalf("error while waiting for deploys: %v", err)
+		require.NoError(t, err, "error while waiting for deploys")
```

Comment:
Optionally, these can be `f.NoError` as we've done mostly everywhere else.

Reply:
I'm updating the ones I'm modifying in this file, but I see lots of inconsistency here, e.g. https://github.com/hashicorp/nomad/blob/v1.0.2/e2e/consul/consul.go#L62-L92 and can come back to them with a cleanup later.

Reply:
🤦 I merged before I pushed the change - will do it in the next PR, sorry!